cdap-ui/app/cdap/components/PipelineList/PipelineList.scss

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+@import '~styles/colors.scss';
+
+$pipeline-view-header: 55px;
+$pipeline-bg-color: $grey-08;
+$view-header-bg-color: white;
+$cog-icon-color: $grey-04;
+$cog-icon-hover-color: $blue-04;
+$popover-width: 100px;
+$delete-color: $red-02;
+$disabled-color: $grey-04;
+
+.pipeline-list {
+  // substracting the height of global header and footer
+  height: calc(100vh - 50px - 53px);
```

Comment:
We need to think about this pattern. Ideally we should just have a margin top and have the footer be at the end of the content. Context for this https://issues.cask.co/browse/CDAP-14418. 

Is it possible to change this here and see how it scales out?

Reply:
footer is globally scoped though, not sure if I can change it just for this particular view?

Reply:
Hhmmm right now its having a fixed position. May be we should change that in a separate PR. It should be positioned static and let it flow based on the content.